docker-compose.yml

```diff
@@ -36,27 +34,32 @@ services:
       POSTGRES_USER: nakadi
       POSTGRES_PASSWORD: nakadi
       POSTGRES_DB: local_nakadi_db
-    network_mode: "host"
 
   zookeeper:
-    image: wurstmeister/zookeeper:3.4.6
-    network_mode: "host"
+    image: zookeeper:3.4.9
+    hostname: zookeeper
     ports:
-      - "2181:2181"
+    - "2181:2181"
+    environment:
+      ZOO_MY_ID: 1
+    volumes:
+    - ./zk-kafka/zookeeper/data:/data
+    - ./zk-kafka/zookeeper/datalog:/datalog
 
   kafka:
-    image: wurstmeister/kafka:1.1.0
-    network_mode: "host"
+    image: confluentinc/cp-kafka:5.0.0
+    hostname: kafka
     ports:
-      - "9092:9092"
-    depends_on:
-      - zookeeper
+    - "9092:9092"
     environment:
-      KAFKA_ADVERTISED_HOST_NAME: localhost
-      KAFKA_ADVERTISED_PORT: 9092
-      KAFKA_ZOOKEEPER_CONNECT: localhost:2181
       KAFKA_AUTO_CREATE_TOPICS_ENABLE: 'false'
       KAFKA_DELETE_TOPIC_ENABLE: 'true'
+      KAFKA_ADVERTISED_LISTENERS: LISTENER_DOCKER_INTERNAL://kafka:19092,LISTENER_DOCKER_EXTERNAL://${DOCKER_HOST_IP:-127.0.0.1}:9092
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: LISTENER_DOCKER_INTERNAL:PLAINTEXT,LISTENER_DOCKER_EXTERNAL:PLAINTEXT
+      KAFKA_INTER_BROKER_LISTENER_NAME: LISTENER_DOCKER_INTERNAL
+      KAFKA_ZOOKEEPER_CONNECT: "zookeeper:2181"
       KAFKA_BROKER_ID: 0
     volumes:
-      - /var/run/docker.sock:/var/run/docker.sock
\ No newline at end of file
+    - /var/run/docker.sock:/var/run/docker.sock
+    depends_on:
+    - zookeeper
```

Comment:
@adyach I think the best option is to add support of EditorConfig. It will solve this kind of problems. What do you think about it? I can create a issue.

Reply:
It makes sense to file an issue for that definitely! For now we can go with the simple fix.